openpower/package/pkg-versions.mk

```diff
@@ -99,9 +101,21 @@ if [ -f $$(OPENPOWER_VERSION_DIR)/$(1).patch.txt ]; then \
 	echo -n "-p" >> $$($(2)_VERSION_FILE); \
 	cat $$(OPENPOWER_VERSION_DIR)/$(1).patch.txt >> $$($(2)_VERSION_FILE); fi \
 fi
+if [ "HOSTBOOT_P10" == "$(2)" ]; then \
```

Comment:
For future-proofing it would be better if this excluded the P8/P9 paths rather than forcing us to change this again in P11, etc.